plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1313,8 +1313,8 @@ protected void setupMemoryBalloonStatsPeriod(Connect conn) {
                 }
                 s_logger.debug(String.format("The memory balloon stats period [%s] has been set successfully for the VM (Libvirt Domain) with ID [%s] and name [%s].",
                         currentVmBalloonStatsPeriod, vmId, dm.getName()));
-            } catch (final LibvirtException e) {
-                s_logger.warn("Failed to set up memory balloon stats period." + e.getMessage());
+            } catch (final Exception e) {
```

Comment:
Is this needed or can we add a specific Exception to the catch clause?

Reply:
@DaanHoogland we are trying to read many other parameters in that parser. so there are chances of other errors such as NPEs, cast errors. While discussing with Wei several other found in IDE, so thought a common exception could catch it and warn the message. 
![image](https://github.com/apache/cloudstack/assets/3348673/d3f0e503-f4eb-4d55-8dc5-6f7d66c37a81)

Reply:
ok, I hate it and I would like to see a message per exception, but no -1 for that

Reply:
:) So you are giving +1 on this right ;)